ape_safe/_cli.py

```diff
@@ -98,22 +98,34 @@ def remove(cli_ctx, alias):
     cls=NetworkBoundCommand, short_help="See pending transactions for a locally-tracked Safe"
 )
 @network_option()
+@click.option("sign_with_local_signers", "--sign", is_flag=True)
+@click.option("--execute", is_flag=False, flag_value=True, default=False)
 @existing_alias_argument(account_type=SafeAccount)
-def pending(network, alias):
+def pending(network, sign_with_local_signers, execute, alias):
     safe = accounts.load(alias)
-    local_signers = set(signer for signer in accounts if signer.address in safe.signers)
-    if local_signers:
-        click.echo("Local Signer(s) detected!")
-        sign_with_local_signers = click.confirm("Do you want to sign unconfirmed transactions")
 
-    else:
-        sign_with_local_signers = False
+    if isinstance(execute, str):
```

Comment:
all this can be placed in the callback of `execute` to keep the cmd implementation focused on execution rather than handling CLI arguments.

(extra, nit)